src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfiguration.swift

```diff
@@ -17,7 +17,7 @@
 // under the License.
 //
 
-import Foundation
+import UIKit
```

Comment:
Hmm not sure if I understand why.

Reply:
Reverted that change now, was needed for a previous approach that we abandoned.